src/vanilla/Model/CodeModelTS.cs

```diff
@@ -39,6 +40,28 @@ public class CodeModelTS : CodeModel
         [JsonIgnore]
         public string PackageName => Settings.Multiapi && !Settings.MultiapiLatest ? ApiVersionPackageName : Settings.PackageName;
 
+        public string BundleFilename
+        {
+            get
+            {
+                string packageName = PackageName ?? "bundle";
+                int slashIndex = packageName.IndexOf("/");
+                string bundleFilename = packageName.Substring(slashIndex + 1);
+                return bundleFilename;
+            }
+        }
+
+        public string BundleVarName
+        {
+            get
+            {
+                string packageName = PackageName ?? "bundle";
+                int slashIndex = packageName.IndexOf("/");
+                string bundleVarName = packageName.Substring(slashIndex + 1).ToPascalCase();
+                return bundleVarName;
```

Comment:
This is almost identical to `BundleFileName`. Can you change this to just be `return BundleFilename.ToPascalCase();`?